Makefile

```diff
@@ -138,4 +141,8 @@ test-e2e-single-node: install-go-junit-report
 	set -o pipefail; go test -tags=$(GOTAGS) -failfast -timeout 120m -v$${WHAT:+ -run="$$WHAT"} ./test/e2e-single-node/ | ./hack/test-with-junit.sh $(@)
 
 bootstrap-e2e: install-go-junit-report install-setup-envtest
-	set -o pipefail; go test -tags=$(GOTAGS) -v$${WHAT:+ -run="$$WHAT"} ./test/e2e-bootstrap/ | ./hack/test-with-junit.sh $(@)
+	set -o pipefail; \
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --remote-bucket openshift-kubebuilder-tools --use-deprecated-gcs --bin-dir $(PROJECT_DIR)/bin)" \
+	echo "KUBEBUILDER_ASSETS set to $$KUBEBUILDER_ASSETS"; \
```

Comment:
I think this line is causing issues, you have `KUBEBUILDER_ASSETS` set here, but then on the next line (because of the semi-colon), you've broken the env var setting and go test is not getting the value set